pkg/registry/apis/provisioning/repository/github_test.go

```diff
@@ -114,15 +114,17 @@ func TestParseWebhooks(t *testing.T) {
 					Enabled: true, // required to accept sync job
 				},
 				GitHub: &provisioning.GitHubRepositoryConfig{
-					Repository: "git-ui-sync-demo",
-					Owner:      "grafana",
-					Branch:     "main",
+					URL:    "https://github.com/grafana/git-ui-sync-demo",
```

Comment:
this one doesn't have the slack as we require in the validation.

Reply:
now it matches... this test does not call the validation so it was fine before